src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+package main.java;
+
+import java.util.Scanner;
+
 public class Duke {
+    public static final int MAX_LIST_SIZE = 100;
+    private static Task[] tasks=new Task[MAX_LIST_SIZE];//user's to-do-list (max 100 items)
```

Comment:
Hi, operators should be surrounded by a space character, just like line 6.
Good use of named constant 'MAX_LIST_SIZE'!